contrib/runners/remote_command_runner/runner.yaml

```diff
@@ -0,0 +1,80 @@
+- aliases:
+  - run-remote
```

Comment:
I removed this, but it had a cascading effect. Would result in a few more files being changed. I changed it back because I think this PR is big enough as it is. Can make this change in another PR.